Marlin/src/module/planner.cpp

```diff
@@ -2041,7 +2041,10 @@ bool Planner::_populate_block(block_t * const block, bool split_move,
   #endif
 
   #if ENABLED(SLOWDOWN)
-    if (WITHIN(moves_queued, 2, (BLOCK_BUFFER_SIZE) / 2 - 1)) {
+    #ifndef SLOWDOWN_DIVISOR
+      #define SLOWDOWN_DIVISOR 2
+    #endif
+    if (WITHIN(moves_queued, 2, (BLOCK_BUFFER_SIZE) / (SLOWDOWN_DIVISOR) - 1)) {
```

Comment:
It might be a benefit to protect us from ourselves and move the `ifndef` into the sanity checks and add a sanity check to make sure it isn't set to 1 (which would be a division by 0 exception?

Because you know it will happen someday :)